homeassistant/components/matter/strings.json

```
@@ -340,6 +340,15 @@
           "paused": "[%key:common::state::paused%]"
         }
       },
+      "esa_op_out_state": {
+        "name": "Energy optimization opt out",
```

Hyphen missing here – the states below have it:

```suggestion
        "name": "Energy optimization opt-out",
```